code/modules/fishing/fish/types/weird.dm

```diff
@@ -0,0 +1,398 @@
+/obj/item/fish/starfish/chrystarfish
```

Comment:
Another thing, the parent type has a subtle emissive appearance that makes it glow in the dark slightly while this has none. That's what it's causing CI issues (as well as a trait that wasn't included in the `globalvars/list/_traits.dm` file.